frontend/src/static/css/config/_dark_theme.scss

```
@@ -365,13 +365,13 @@ body.dark_theme {
 
 	/* ################################################## */
 
-	--user-action-form-inner-bg-color: #242424;
+	--user-action-form-inner-bg-color: var(--cinemata-pacific-deep-900);
 	// --user-action-form-inner-title-border-bottom-color: rgba(255, 255, 255, 0.1);
```

Stylelint (17.11.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)